drivers/led_strip/Kconfig.ws2812b_sw

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WS2812B_SW
+	bool "Enable WS2812B software-based LED strip driver"
+	# Only a Cortex-M0 inline assembly implementation is supported for now
```

Comment:
Is the implementation dependent on CPU clock speed? If so, state it or we need a way for the driver to be more portable across CPU speeds.

Reply:
Yes, it's dependent on the speed, and I wouldn't expect each CPU/architecture to have their own inline assembly snippet. I'll add a comment to state the requirement, but I think it should be in the code itself rather than here in the Kconfig.

Reply:
Just another note regarding CPU speed, based on what I've found when doing research online, it seems the minimum theoretical CPU speed that should be able to accomplish the signal generation task is about 4 MHz.